web/cypress/integration/classifications/classifications.spec.js

```diff
@@ -1,151 +1,126 @@
-            newValueOfPriority = parseFloat(newValueOfPriority);
-            expect(newValueOfPriority).to.eq(oldPriorityValue + 1);
-          });
-      });
-  });
 
   it('should be possible to edit the Category of a classification', () => {
     cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
     cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
 
-    cy.get('.required > .dropdown > .btn').click();
-
+    cy.get('ng-form').find('mat-form-field').find('mat-select[role="listbox"]').click();
+ 
     cy.wait(Cypress.env('dropdownWait'));
 
-    cy.get('.dropdown-menu.show > li').contains('PROCESS').click();
-
-    cy.get('[title="Save"] > .material-icons').click();
-
-    cy.reload();
-
-    cy.wait(Cypress.env('pageReload'));
-
+    cy.get('mat-option').contains('PROCESS').click();
+    cy.get('button').contains('Save').click();
+    
+    cy.reloadPageWithWait();
+   
     // assure that its process now
-
-    cy.get('.required > .dropdown > .btn').contains('PROCESS').should('be.visible');
-
+    cy.get('ng-form').find('mat-form-field').find('mat-select[role="listbox"]').contains('PROCESS').should('be.visible');
     // change back to external
-
-    cy.get('.required > .dropdown > .btn').click();
+    cy.get('ng-form').find('mat-form-field').find('mat-select[role="listbox"]').click();
 
     cy.wait(Cypress.env('dropdownWait'));
 
-    cy.get('.dropdown-menu.show > li').contains('EXTERNAL').should('be.visible').click();
-
-    cy.get('[title="Save"] > .material-icons').click();
+    cy.get('mat-option').contains('EXTERNAL').should('be.visible').click();
+    cy.get('button').contains('Save').click();
   });
 
+  
   it('should be possible to edit the Description of a classification', () => {
     cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
+    cy.wait(Cypress.env('pageReload'));
     cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
 
     const editedValue = 'CY-TEST-DESC';
-
     cy.get('#classification-description').clear().type(editedValue);
+    cy.get('button').contains('Save').click();
 
-    cy.get('[title="Save"] > .material-icons').click();
-
-    cy.reload();
+    cy.reloadPageWithWait();
 
     cy.wait(Cypress.env('pageReload'));
-
     cy.get('#classification-description').should('have.value', editedValue);
   });
 
-  it('should be possible to edit the Service Level of a classification', () => {
-    cy.visit(Cypress.env('appUrl') + Cypress.env('adminUrl') + '/classifications');
-    cy.contains(Cypress.env('testValueClassificationSelectionName')).click();
-
-    const editedValue = 'P99D';
-
-    cy.get('#classification-service-level').clear().type(editedValue);
-
-    cy.get('[title="Save"] > .material-icons').click();
-
-    cy.reload();
-
-    cy.wait(Cypress.env('pageReload'));
-
-    cy.get('#classification-service-level').should('have.value', editedValue);
-  });
```

Comment:
Why did you remove this test?

Reply:
This test still exists and has not been removed.